Header header
uint16 id
float32 x
float32 y
float32 z
float32 x_distance
float32 y_distance
float32 z_distance
